ksql-engine/src/main/java/io/confluent/ksql/structured/SelectValueMapper.java

```diff
@@ -47,6 +47,9 @@
 
   @Override
   public GenericRow apply(final GenericRow row) {
+    if (row == null) {
```

Comment:
Wouldn't it be better to emit a GenericRow where each field is null? What would happen if I had a query like:
`SELECT foo, bar from FOOBAR WHERE foo != null` and the topic has a null record? I think the current impl will raise a NullPointerException trying to evaluate the WHERE expression.